configs/AM65X/AM65X_linux_toc.txt

```diff
@@ -53,8 +53,9 @@ linux/Foundational_Components/Kernel/Kernel_Drivers/DCAN
 linux/Foundational_Components/Kernel/Kernel_Drivers/Display/DSS7
 linux/Foundational_Components/Kernel/Kernel_Drivers/GPIO
 linux/Foundational_Components/Kernel/Kernel_Drivers/I2C
-linux/Foundational_Components/Kernel/Kernel_Drivers/Network/PRUSS
 linux/Foundational_Components/Kernel/Kernel_Drivers/Network/CPSW2g
+linux/Foundational_Components/Kernel/Kernel_Drivers/Network/CPSW-EST
```

Comment:
This file does not have any references to AM65, unlike CPSW2g and CPSW-IET.

Just want to confirm this is required for AM65, right?

Reply:
@danish-ti

Reply:
@StaticRocket IET and EST both apply to AM65x as well. CPSW in AM65x supports both IET and EST. The documentation is written with AM62 and J7VCL being the DUTs, however the same are applicable to all other DUTs having CPSW. 

@cshilwant  Pls merge this if there are no more comments.